cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/actor/ProgressiveSolver.scala

```diff
@@ -133,7 +136,11 @@ class ProgressiveSolver(val dataManager: ActorRef,
             "end" -> JsNumber(boundary.getEnd().getMillis())
         ))
         // for query with slicing request, add current timeInterval information in its query results.
-        val infoValue = queryGroup.postTransform.transform(JsArray(mergedResults))
+        //val infoValue = queryGroup.postTransform.transform(JsArray(mergedResults))
+
+        val newresult = QueryPlanner.handleAvg(mergedResults)
```

Comment:
```suggestion
        val avgHandledResults = QueryPlanner.handleAvg(mergedResults)
```
Let's use more informative variable names.